daily_open_prs.rb

```diff
@@ -1,99 +1,83 @@
 #!/usr/bin/env ruby
-#This script is to output a csv file to tally how many PRs are currently open on each day between two date ranges. It is also split into both PRs raised by community members and PRs raised by Puppet members.
+# frozen_string_literal: true
+
+# This script is to output a csv file to tally how many PRs are currently open on each day between two date ranges. It is also split into both PRs raised by community members and PRs raised by Puppet members.
 
 require 'optparse'
 require 'csv'
 require 'octokit'
 require_relative 'octokit_utils'
+require 'json'
+
+output = File.read('modules.json')
```

Comment:
To make this at least nominally useful to folks outside the company, please make the loaded file a command line option. It can default to this value.